charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,14 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+      - name: ghostunnel-server
+        command: ["ghostunnel"]
+        args: ["server", "--listen", "localhost:8443", "--target", "localhost:9083", "--key", "/opt/hive/tls/tls.key", "--cert", "/opt/hive/tls/tls.crt", "--cacert", "/opt/hive/tls/ca.crt", "--allow-cn", "Any"]
```

Comment:
we could also use an environment variable to control the location of the key, cert, CA cert instead of hardcoding the values in the argument array

Reply:
Also prefer a hyphen based list for long list of args. Eg:

```
args:
- server
- --listen
- localhhost:8443
- --target
- localhost:9083
....
```
etc
```